pkg/mount/mount.go

```diff
@@ -78,17 +80,25 @@ func RecursiveUnmount(target string) error {
 		}
 		logrus.Debugf("Trying to unmount %s", m.Mountpoint)
 		err = Unmount(m.Mountpoint)
```

Comment:
In here, `Unmount()` calls `Mounted()`, which is expensive, and making optimization of removing another `Mounted()` on an error path kinda moot.

Perhaps `unmount()` should be used here instead, as it does not call `Mounted()`.